ophyd/areadetector/plugins.py

```diff
@@ -785,6 +785,7 @@ class FilePlugin(PluginBase, GenerateDatumInterface, version=(1, 9, 1), version_
     auto_increment = Cpt(SignalWithRBV, 'AutoIncrement', kind='config')
     auto_save = Cpt(SignalWithRBV, 'AutoSave', kind='config')
     capture = Cpt(SignalWithRBV, 'Capture')
+    create_directory_depth = Cpt(SignalWithRBV, suffix="CreateDirectory")
```

Comment:
We already have this pv, but it is in the class for 2-2+ ADs (see https://github.com/bluesky/ophyd/blob/949efc0fa560a4affc26482276177f26daf7d4e8/ophyd/areadetector/plugins.py#L1042-L1047 )

Reply:
Here's my setup.  Why didn't I get it then?

```In [1]: 
   ...: class MySingleTriggerSimDetector(SingleTrigger, SimDetector): 
   ...:        
   ...:     image = Component(ImagePlugin, suffix="image1:")
   ...:     hdf1 = Component(
   ...:         MyHDF5Plugin,
   ...:         suffix='HDF1:', 
   ...:         root='/',                               # for databroker
   ...:         
   ...:         # note: path MUST, must, MUST have trailing "/"!!!
   ...:         #  ...and... start with the same path defined in root (above)
   ...:         write_path_template="/tmp/simdet/%Y/%m/%d/",    # for EPICS AD
   ...:     )
   ...:     tiff1 = Component(
   ...:         MyTIFFPlugin,
   ...:         suffix='TIFF1:', 
   ...:         write_path_template="/tmp/simdet/%Y/%m/%d/",    # for EPICS AD
   ...:     )
   ...: 
   ...: 
   ...: _ad_prefix = "vm7SIM1:"        # IOC prefix

In [2]: adsimdet = MySingleTriggerSimDetector(_ad_prefix, name='adsimdet')

In [3]: hasattr(adsimdet.tiff1, "create_directory")
Out[3]: False

In [5]: TIFFPlugin??
Init signature: TIFFPlugin(*args, **kwargs)
Docstring:      AreaDetector plugin base class
Source:        
class TIFFPlugin(FilePlugin):
    _default_suffix = 'TIFF1:'
    _suffix_re = 'TIFF\d:'
    _html_docs = ['NDFileTIFF.html']
    _plugin_type = 'NDFileTIFF'
File:           ~/Apps/anaconda/envs/bluesky/lib/python3.6/site-packages/ophyd/areadetector/plugins.py
Type:           ComponentMeta
```

Need a custom TIFFPlugin that is otherwise identical except subclass of `FilePlugin_V22`?

Reply:
Aha.  That feature is in a PR, not a release.  That's why I can't find it locally.